content/docs/dvclive/dvclive-with-dvc.md

```diff
@@ -80,18 +81,25 @@ $ dvc repro train
 After that's finished, you should see the following content in the project:
 
 ```dvc
-$ ls
-dvc.lock  training_metrics       train.py
-dvc.yaml  training_metrics.json
+$ tree
```

Comment:
Minor/general note: BTW I started noticing all these Linux-specific `$ tree` commands. You could omit them if it's clear that the block represents a directory structure (in most cases that's the case in my experience).